header/core/listener.go

```diff
@@ -21,24 +22,30 @@ import (
 // broadcasts the new `ExtendedHeader` to the header-sub gossipsub
 // network.
 type Listener struct {
-	bcast     libhead.Broadcaster[*header.ExtendedHeader]
-	fetcher   *core.BlockFetcher
-	bServ     blockservice.BlockService
+	fetcher *core.BlockFetcher
+	bServ   blockservice.BlockService
+
 	construct header.ConstructFn
-	cancel    context.CancelFunc
+
+	extHeaderBroadcaster libhead.Broadcaster[*header.ExtendedHeader]
+	edsHashBroadcaster   shrexsub.BroadcastFn
```

Comment:
```suggestion
	headerBroadcaster libhead.Broadcaster[*header.ExtendedHeader]
	hashBroadcaster   shrexsub.BroadcastFn
```

Reply:
We already have Ext mentioning in the type param

Reply:
done